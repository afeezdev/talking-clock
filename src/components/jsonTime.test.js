import { shallow } from 'enzyme';
import React from 'react';
import JsonTime from './jsonTime';

it('expect to render JsonTime component' , () => {

  expect(shallow(<JsonTime />)).toMatchSnapshot()
})