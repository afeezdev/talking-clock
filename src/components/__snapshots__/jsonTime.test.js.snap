// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render JsonTime component 1`] = `ShallowWrapper {}`;
